internal/api/api.go

```diff
@@ -26,8 +26,14 @@ import (
 
 const (
 	// ListenerAddrKey is the environment variable for the Octant listener address.
-	ListenerAddrKey  = "listener-addr"
+	ListenerAddrKey = "listener-addr"
+
+	// AcceptedHostsKey the list of accepted hosts to be allowed to connect over web sockets
 	AcceptedHostsKey = "accepted-hosts"
+
+	// AcceptLocalIPKey if no accepted hosts are specified this flag will add all the local IP addresses
+	//starting with `192.168.1.` to the accepted hosts
```

Comment:
This is too specific, anecdotally, I use `192.168.2.`

Reply:
sounds good to me